lib/aasm/persistence/active_record_persistence.rb

```diff
@@ -189,8 +191,10 @@ def aasm_fire_event(state_machine_name, name, options, *args, &block)
             end
 
             if options[:persist] && success
-              event.fire_callbacks(:after_commit, self, *args)
-              event.fire_global_callbacks(:after_all_commits, self, *args)
+              # Delegating to ActiveRecord
```

Comment:
Super neat stuff. In my humble opinion, perhaps it's just better to remove the lines below vs comment out.